lib/src/Base/Func/openturns/BasisSequenceFactoryImplementation.hxx

```diff
@@ -88,6 +88,10 @@ public:
   /** Selection history accessor */
   virtual Collection<Indices> getSelectionHistory(Collection<Point> & coefficientsHistory) const;
 
+  /** Verbose accessor */
+  Bool getVerbose() const;
+  void setVerbose(const Bool verbose);
```

Comment:
we just removed all the verbose booleans, just choose a log level instead

Reply:
Fixed at e7d9d5287.